com.unity.render-pipelines.high-definition/Runtime/Compositor/AlphaInjection.cs

```diff
@@ -0,0 +1,61 @@
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.HighDefinition;
+using System;
+
+namespace UnityEngine.Rendering.HighDefinition.Compositor
+{
+    [Serializable, VolumeComponentMenu("Post-processing/Custom/AlphaInjection")]
```

Comment:
Please add "preview here"

Reply:
and for the other one as well

Reply:
Even better, there was no need to expose the custom pass in the menu (is controlled by the compositor), so I removed the entry.